google-news-reader/google-news-reader/ViewController.swift

```diff
@@ -13,6 +13,14 @@ class ViewController: UIViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
         // Do any additional setup after loading the view, typically from a nib.
+        
+        NetworkManager().fetchAllArticlesWithCompletion { (data, error) -> Void in
+            if let unwrappedData = data as? NSData {
+                NetworkManager().parseAllArticleData(unwrappedData, completion: { (content, error) -> Void in
+                    print(content)
```

Comment:
For testing/debugging only.  Ultimately I want to chain the `NetworkManager` methods together so that a View Controller is not manually calling the methods to fetch and parse the data.